### DISCUSSION TITLE
How to send and receive custom binary data?

### DISCUSSION THREAD

**Discussion by congchenutd** [HaishinKit/HaishinKit.swift#1596] [Idea]
E.g., can I send my own data in real-time using a separate stream/connection? I don't see a public API for it. I know it's simple to do it in my own code, but I figure it might be convenient if HaishinKit provides such an API.

**Comment by shogo4405** [comment]
I probably haven’t provided that feature. For what purpose do you need such functionality? If you can explain in more detail what you want to do, I might be able to give you a better answer.
